### ISSUE TITLE
Getting "undefined: undefined" when using enums

### ISSUE THREAD

**Issue by SorenHolstHansen** [AlecAivazis/houdini#166]
I have a type-graphql class like this
```
import { registerEnumType } from 'type-graphql';

enum Color {
	black = 'black',
	white = 'white',
	gray = 'gray',
	orange = 'orange',
	purple = 'purple',
	red = 'red',
	yellow = 'yellow',
	green = 'green',
	blue = 'blue',
}

registerEnumType(Color, {
	name: 'Color', // this one is mandatory
	description: 'Various Colors', // this one is optional
});

@ObjectType()
class Grid {
        // ...
	@Field(() => [[Color]], { nullable: 'items' })
	colors?: (Color | null)[][];
        // ...
}

@Resolver(Sudoku)
export class GridResolver {
    @Query(() => [Grid])
    getGrids(): Grid[] {
    // return array of grids from db
    }
}
```
And I can successfully query it with the graphIQL playground. However when I query it with houdini like this
```
<script lang="ts">
    import { query, graphql, GetGrids } from '$houdini';

    const { data } = query<GetGrids>(graphql`
		query GetGrids {
			getGrids {
				colors
			}
		}
	`);
    console.log("data", $data);
</script>
```
I get
```
data {
    getGrids: [
        // ...
        { undefined: undefined }, // should be colors: ["green", ...]
        // ...
    ]
}
```
instead of the colors.

Am I doing something wrong?

**Comment by AlecAivazis** [comment]
Hey @SorenHolstHansen - sorry you are running into issues. Can you share your component code?

**Comment by SorenHolstHansen** [comment]
I have updated my issue with the info

**Comment by AlecAivazis** [comment]
Would you be able to push this up somewhere i can look at?

**Comment by pixelmund** [comment]
I remember something like this happened to me too.

**Comment by AlecAivazis** [comment]
That's good to know. @SorenHolstHansen do you mind sharing your queries artifact aswell? I have a test for reading and writing enum values so it'd be helpful to get some more info so i can figure out what the difference is

**Comment by SorenHolstHansen** [comment]
Have a look at https://github.com/SorenHolstHansen/houdini-enums-error. This should recreate the error. The error should happen for general enums, not just inside nested arrays.

**Comment by AlecAivazis** [comment]
Awesome thanks! I was able to reproduce this and have a fix. Will open a PR up shortly

**Comment by AlecAivazis** [comment]
This should be fixed in `0.10.7`

**Comment by SorenHolstHansen** [comment]
Thanks man!
